internal/router/api/v1/telemetry.go

```diff
@@ -0,0 +1,71 @@
+/*
+Copyright (C) 2021-2023, Kubefirst
+
+This program is licensed under MIT.
+See the LICENSE file for more details.
+*/
+package api
+
+import (
+	"net/http"
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/kubefirst/kubefirst-api/internal/db"
+	"github.com/kubefirst/kubefirst-api/internal/telemetryShim"
+	"github.com/kubefirst/kubefirst-api/internal/types"
+	log "github.com/sirupsen/logrus"
+)
+
+// PostTelemetry godoc
+// @Summary Create a Telemtry Event
+// @Description Create a Telemtry Event
+// @Tags cluster
+// @Accept json
+// @Param	cluster_name	path	string	true	"Cluster name"
+// @Param	definition	body	types.ClusterDefinition	true	"Cluster create request in JSON format"
+// @Success 202 {object} types.TelemetryRequest
```

Comment:
The telemetry request should actually go in the params section, the response should follow the typical success response